### PR TITLE
[Kaptain] Tutorial legal Attributions

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

No JIRA

## Description of changes being made

Added legal attributions for Kaptain tutorial documentation that uses minIO.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4486.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by lukeogg** [comment]
I believe these will need to be changed in the source and synced rather than edited directly.

**Comment by gvasquezvargas** [comment]
@lukeogg do we need to change the wording from "This Docker image" to "This Helm chart" here as well?

File: pages/dkp/kaptain/1.3.0/tutorials/sdk/pytorch/index.md [hunk 1]
```diff
@@ -722,3 +722,5 @@ curl --location \
     {"predictions": [{"7": 0.9999998807907104, "2": 5.1130371048202505e-08, "3": 3.742747978208172e-08, "9": 2.259610054622385e-09, "1": 1.1725632687031862e-09}]}
 
 We can see that the class with label "7" has the largest probability; the neural network correctly predicts the image to be a number 7.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z) and [https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z](https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z).
```
Comment (lukeogg) [inline pages/dkp/kaptain/1.3.0/tutorials/sdk/pytorch/index.md:726]:
```suggestion
This tutorial image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z) and [https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z](https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z).
```

File: pages/dkp/kaptain/1.3.0/tutorials/sdk/quick-start/index.md [hunk 2]
```diff
@@ -529,3 +529,5 @@ curl --location \
 
 
 We can see that the class with label "7" has the largest probability; the neural network correctly predicts the image to be a number 7.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z) and [https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z](https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z).
```
Comment (lukeogg) [inline pages/dkp/kaptain/1.3.0/tutorials/sdk/quick-start/index.md:533]:
```suggestion
This tutorial image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z) and [https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z](https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z).
```

File: pages/dkp/kaptain/1.3.0/tutorials/sdk/tensorflow/index.md [hunk 3]
```diff
@@ -523,3 +523,5 @@ curl --location \
     }
 
 The last class has the largest probability; the neural network correctly predicts the image to be a 9.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z) and [https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z](https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z).
```
Comment (lukeogg) [inline pages/dkp/kaptain/1.3.0/tutorials/sdk/tensorflow/index.md:527]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.3.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z) and [https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z](https://github.com/minio/minio/tree/RELEASE.2021-07-30T00-02-00Z).
```

File: pages/dkp/kaptain/2.0.0/tutorials/pipelines/index.md [hunk 4]
```diff
@@ -738,3 +738,5 @@ jq -M --exit-status '.predictions[0] | indices(max)[0] == 9' output.json
 
 
 For more details on the URL, please check out this [example](https://github.com/kserve/kserve/tree/master/docs/samples/v1beta1/tensorflow#run-a-prediction).
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.0.0/tutorials/pipelines/index.md:742]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/2.0.0/tutorials/sdk/pytorch/index.md [hunk 5]
```diff
@@ -727,3 +727,5 @@ curl --location \
     {"predictions": [{"7": 0.9999998807907104, "2": 5.1130371048202505e-08, "3": 3.742747978208172e-08, "9": 2.259610054622385e-09, "1": 1.1725632687031862e-09}]}
 
 We can see that the class with label "7" has the largest probability; the neural network correctly predicts the image to be a number 7.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.0.0/tutorials/sdk/pytorch/index.md:731]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/2.0.0/tutorials/sdk/quick-start/index.md [hunk 6]
```diff
@@ -531,3 +531,5 @@ curl --location \
 
 
 We can see that the class with label "7" has the largest probability; the neural network correctly predicts the image to be a number 7.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.0.0/tutorials/sdk/quick-start/index.md:535]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/2.0.0/tutorials/sdk/tensorflow/index.md [hunk 7]
```diff
@@ -525,3 +525,5 @@ curl --location \
     }
 
 The last class has the largest probability; the neural network correctly predicts the image to be a 9.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.0.0/tutorials/sdk/tensorflow/index.md:529]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/2.1.0/tutorials/pipelines/index.md [hunk 8]
```diff
@@ -738,3 +738,5 @@ jq -M --exit-status '.predictions[0] | indices(max)[0] == 9' output.json
 
 
 For more details on the URL, please check out this [example](https://github.com/kserve/kserve/tree/master/docs/samples/v1beta1/tensorflow#run-a-prediction).
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.1.0/tutorials/pipelines/index.md:742]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/2.1.0/tutorials/sdk/image-builder/index.md [hunk 9]
```diff
@@ -453,3 +453,5 @@ model.train(
 ```
 
 To learn more about creating and deploying machine learninig models with Kaptain SDK, refer to [Kaptain SDK with Tensorflow](../tensorflow) or [Kaptain SDK with Pytorch](../pytorch) tutorials.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.1.0/tutorials/sdk/image-builder/index.md:457]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/2.1.0/tutorials/sdk/quick-start/index.md [hunk 10]
```diff
@@ -531,3 +531,5 @@ curl --location \
 
 
 We can see that the class with label "7" has the largest probability; the neural network correctly predicts the image to be a number 7.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.1.0/tutorials/sdk/quick-start/index.md:535]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/2.1.0/tutorials/sdk/tensorflow/index.md [hunk 11]
```diff
@@ -525,3 +525,5 @@ curl --location \
     }
 
 The last class has the largest probability; the neural network correctly predicts the image to be a 9.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/2.1.0/tutorials/sdk/tensorflow/index.md:529]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.1.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

File: pages/dkp/kaptain/1.2.0/tutorials/sdk/tensorflow/index.md [hunk 12]
```diff
@@ -512,3 +512,5 @@ curl --location \
     }
 
 The last class has the largest probability; the neural network correctly predicts the image to be a 9.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at this URL: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/1.2.0/tutorials/sdk/tensorflow/index.md:516]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at this URL: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z)
```

File: pages/dkp/kaptain/1.2.0/tutorials/sdk/pytorch/index.md [hunk 13]
```diff
@@ -711,3 +711,5 @@ curl --location \
     {"predictions": [{"7": 0.9999998807907104, "2": 5.1130371048202505e-08, "3": 3.742747978208172e-08, "9": 2.259610054622385e-09, "1": 1.1725632687031862e-09}]}
 
 We can see that the class with label "7" has the largest probability; the neural network correctly predicts the image to be a number 7.
+
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at this URL: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z)
```
Comment (lukeogg) [inline pages/dkp/kaptain/1.2.0/tutorials/sdk/pytorch/index.md:715]:
```suggestion
This tutorial includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 1.2.0 are available at this URL: [https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z](https://github.com/minio/minio/tree/RELEASE.2020-12-03T05-49-24Z)
```

File: pages/dkp/kaptain/2.1.0/tutorials/pipelines/index.md (hunk 8 above)
Comment (lukeogg) [reply pages/dkp/kaptain/2.1.0/tutorials/pipelines/index.md:742]: With the current version, we need to update the source notebooks as well - these changes will get overwritten when we sync. I think you (@gvasquezvargas ) already created a ticket for that?

**Review by lukeogg — APPROVED** [review]
With one small wording change - LGTM!

**Review by gvasquezvargas — APPROVED** [review]

**Review by Colin1841 — APPROVED** [review]

**Pull request merged**